### PR TITLE
satellite: added ability to inject stripe public key post build

### PR DESCRIPTION
**Pull request by Qweder93**
What: added ability to inject stripe public key post build

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/store/modules/appState.ts [hunk 1]
```diff
@@ -199,5 +204,11 @@ export const appStateModule = {
         [APP_STATE_ACTIONS.SET_SATELLITE_NAME]: function ({commit}: any, satelliteName: string): void {
             commit(APP_STATE_MUTATIONS.SET_NAME, satelliteName);
         },
+        [APP_STATE_ACTIONS.SET_STRIPE_PK]: function ({commit}: any, stripePublicKey: string): void {
+            commit(APP_STATE_MUTATIONS.SET_STRIPE_PK, stripePublicKey);
```
Comment (rikysya) [inline web/satellite/src/store/modules/appState.ts]: we don't need stripe pub key in app state

File: web/satellite/src/components/account/billing/paymentMethods/StripeInput.vue [hunk 2]
```diff
@@ -34,7 +34,7 @@ export default class StripeInput extends Vue {
             return;
         }
 
-        this.stripe = window['Stripe'](process.env.VUE_APP_STRIPE_PUBLIC_KEY);
+        this.stripe = window['Stripe'](this.$store.state.appStateModule.stripePublicKey);
```
Comment (rikysya) [inline web/satellite/src/components/account/billing/paymentMethods/StripeInput.vue]: lets wrap into a class that works with stripe library

Comment (NikolaiYurchenko) [reply web/satellite/src/components/account/billing/paymentMethods/StripeInput.vue]: Agree with @rikysya

File: web/satellite/src/store/modules/appState.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { APP_STATE_ACTIONS } from '@/utils/constants/actionNames';
 import { AppState } from '@/utils/constants/appStateEnum';
 
 import { APP_STATE_MUTATIONS } from '../mutationConstants';
+import {User} from "@/types/users";
```
Comment (NikolaiYurchenko) [inline web/satellite/src/store/modules/appState.ts]: You should run "npm run lint" before commit

File: web/satellite/src/store/modules/appState.ts [hunk 4]
```diff
@@ -199,5 +204,11 @@ export const appStateModule = {
         [APP_STATE_ACTIONS.SET_SATELLITE_NAME]: function ({commit}: any, satelliteName: string): void {
             commit(APP_STATE_MUTATIONS.SET_NAME, satelliteName);
         },
+        [APP_STATE_ACTIONS.SET_STRIPE_PK]: function ({commit}: any, stripePublicKey: string): void {
+            commit(APP_STATE_MUTATIONS.SET_STRIPE_PK, stripePublicKey);
+        },
+    },
+    getters: {
+        stripePublicKey: (state: any): string => state.stripePublicKey,
```
Comment (NikolaiYurchenko) [inline web/satellite/src/store/modules/appState.ts]: useless getter

File: web/satellite/src/utils/metaContentExtractor.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+export class MetaRepository {
```
Comment (NikolaiYurchenko) [inline web/satellite/src/utils/metaContentExtractor.ts]: Add tests

Comment (crawter) [inline web/satellite/src/utils/metaContentExtractor.ts]: we should find better name for this file and for this class. Let's think what intention this class has

File: web/satellite/src/utils/metaContentExtractor.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    public static getMetaContent(metaName: string) {
+        const meta = document.querySelector(`meta[name='${metaName}']`);
+
+        if (meta) {
+            return  meta.getAttribute('content');
```
Comment (crawter) [inline web/satellite/src/utils/metaContentExtractor.ts]: extra space. You should run npm run lint command

File: web/satellite/src/store/mutationConstants.ts [hunk 7]
```diff
@@ -27,6 +27,7 @@ export const APP_STATE_MUTATIONS = {
     CHANGE_STATE: 'CHANGE_STATE',
     TOGGLE_PAYMENT_SELECTION: 'TOGGLE_PAYMENT_SELECTION',
     SET_NAME: 'SET_NAME',
+    SET_STRIPE_PK: 'SET_STRIPE_PK',
```
Comment (crawter) [inline web/satellite/src/store/mutationConstants.ts]: not sure that we need it in our store. Stripe module could store this data

File: satellite/console/consoleweb/server.go [hunk 8]
```diff
@@ -76,6 +76,8 @@ type Config struct {
 type Server struct {
 	log *zap.Logger
 
+	stripePublicKey string
```
Comment (crawter) [inline satellite/console/consoleweb/server.go]: i would place this field in the bottom, cause its not so important as others

**Pull request merged**